components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/internal/impl/UserSessionManagementServiceImpl.java

```diff
@@ -80,11 +81,11 @@ private void validate(String username, String userStoreDomain, String tenantDoma
         }
     }
 
+    // This method invokes only for the local users to resolve the user's unique user id from username.
```

Comment:
Shall we add a more descriptive comment for the function? Since the function is to retrieve the sessions, move the user-id related comment near the logic resolving the user-id